### DISCUSSION TITLE
server / drive setup

### DISCUSSION THREAD

**Discussion by totlsota** [erigontech/erigon#3788] [Q&A] [answered]
Are NVMe drives required for good performance, or are standard SSDs OK?

Min server requirements suggest >=3TB (of presumably SSD?). Discussions here otherwise indicate 2TB would plausibly run for at least 1y-2y perhaps.

If NVMe is the way to go for good performance, then I may try to press for 2TB of space, but wanted some guidance of how flawed that might be before proceeding.

Any comments appreciated.

**Comment by AskAlexSharov** [comment, marked as answer]
ssd is ok
you also can use NVMe for initial sync and when it's full just move Erigon's folder to SSD
